rfc11720

```diff
@@ -92,6 +139,8 @@ The objectives of the myTeams project RFC is to allow communication between myTe
     - Forbidden
 - 14
     - Not found
+- 15
+    - Logout or Lost Connexion
```

Comment:
The server can't send this code if the client lost connection

Reply:
It should be up to the client to detect if the connection to the server is broken